### PR TITLE
I added comment

### PR DESCRIPTION
**Pull request by fatihcam46**

### CODE REVIEW

**Review by ecaliskanunal — APPROVED** [review]

File: src/test/java/Practice_1.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 public class Practice_1 {
   // From git-hub
   // group-14
-  //
+  //I came Fath
```
Comment (ademcigdem) [inline src/test/java/Practice_1.java:4]: FAtih please do proper coding

**Review by ademcigdem — CHANGES_REQUESTED** [review]

**Comment by ecaliskanunal** [comment]
Better to edit your code.

**Review by GokceKochan — APPROVED** [review]
bravo Fatih

File: src/test/java/Practice_1.java (hunk 1 above)
Comment (fatihcam46) [reply src/test/java/Practice_1.java:4]: Yes always

**Review by ecaliskanunal — APPROVED** [review]

**Review by busragokner — APPROVED** [review]
Good

**Review by fikretellek — APPROVED** [review]

**Review by AndrianHrisotimos — APPROVED** [review]
working

**Pull request closed without merging**